src/NServiceBus.Msmq.AcceptanceTests/SubscriptionStorage/When_subscription_exists_for_tx_endpoint.cs

```diff
@@ -69,6 +69,8 @@ public Publisher()
                 });
             }
 
+            // Disable obsolete warning until MessageEndpointMappings has been removed from config and we can replace with code api
+#pragma warning disable CS0612, CS0619, CS0618
```

Comment:
No, this is leftovers from the mass-coverage attempt before the treataserror property was used